### DISCUSSION TITLE
OpenShift 4.7 Disconnected/Offline Installation:

### DISCUSSION THREAD

**Discussion by Jhansi-Rajput** [okd-project/okd#847] [Q&A]
Installing OKD4.7 (4.7.0-0.okd-2021-05-22-050008) in a restricted environment/network. Bootstrap node is accessible and able to pull the images from the mirror registry. When I shoot the bootstrap-complete installation cmd. API came up and running but failed to connect to master nodes. I have added all the ports required for the offline setup still unable to establish the connection to master nodes. 

It would be great if someone can assist.

**Comment by vrutkovs** [comment]
Looks similar to https://github.com/openshift/okd/issues/783 - which FCOS are you using on masters/workers?
